pkgs/servers/mqtt/mosquitto/default.nix

```diff
@@ -12,6 +12,7 @@
 , openssl
 , withSystemd ? lib.meta.availableOn stdenv.hostPlatform systemd
 , systemd
+, uthash
```

Comment:
Why?

Reply:
As far as I see in nixpkgs we always try not to use bundled dependencies if possible.

Reply:
Include this explanation on commit message.

Reply:
added